packages/swarm-components/src/testUtils/screenRenderer.js

```diff
@@ -26,24 +29,36 @@ class screenRenderer {
 	}
 
 	async init() {
-		const { port, staticPath } = this.config;
+		const { port, host } = this.config;
 
 		this.browser = await puppeteer.launch();
 		this.log('Browser is running');
 
 		this.server = Hapi.server({
 			port,
+			host,
 		});
 
 		await this.server.register(inert);
 
-		// serving static files
+		// serving static styles from swarm-styles
 		this.server.route({
 			method: 'GET',
 			path: '/static/{param*}',
 			handler: {
 				directory: {
-					path: Path.join(__dirname, staticPath),
+					path: Path.join(__dirname, '../../../swarm-styles/dist'),
+				},
+			},
+		});
+
+		// serving font files from swarm-docs
```

Comment:
Not sure I follow this. Why are fonts being served from swarm-docs?

Reply:
Oh, I see, because it's the only place that we have the files saved in this repo. So, as a follow up we should find a central, shared location for the font files... potentially even a location that can be used across `mup-web`, `pro-web`, MWC, swarm-ui etc. From a semantics standpoint, I get this works but it's odd.

Reply:
This is odd. The fonts are bundled separately in out apps for caching, so nothing from swarm is consumed. It didn't make sense to have many versions of the assets in the same repo.